### DISCUSSION TITLE
Vocal Suffixes

### DISCUSSION THREAD

**Discussion by THEkatinamicrowave** [CodenameCrew/CodenameEngine#400] [Q&A]
I'm not sure if it's an error or just short-sighted code, but I'm getting a strange error whenever I load this song.

Here's some context:
This is what is inside my song folder
![image](https://github.com/user-attachments/assets/2ae0df41-ccc2-4b2e-8f6c-e9ec5dff37fb)
I always compile my game before running it (I hate navigating through the export folder) and when I load the song in the chart editor, this happens.
![image](https://github.com/user-attachments/assets/29fd62c9-9e67-49a4-b01b-24dca691e4de)
Probably just a coding oversight, but how do I fix this without asking you to fix the engine? Please lmk

P.S: Sorry for making like four discussions within one week; I wanted to spread them out based on specific topic that way the responses weren't ultra confused and talking about different problems every time

**Comment by NeeEoo** [comment]
Maybe just join the discord server, or make github issues

**Reply by THEkatinamicrowave** [reply]
oh yeah issues exist true
as for Discord, my parents won't let me use it yet. I might though when I get the ability
